build.sbt

```diff
@@ -283,6 +287,14 @@ lazy val nativeServer: Project = (project in file("native-server"))
     dockerEntrypoint := Seq("/sbin/tini", "--", "/opt/docker/bin/elasticmq-native-server", "-Dconfig.file=/opt/elasticmq.conf"),
     dockerUpdateLatest := true,
     dockerExposedPorts := Seq(9324,9325),
+    dockerBuildOptions := dockerBuildOptions.value :+ "--platform=linux/arm64,linux/amd64" :+ "--push",
+    dockerBuildCommand := {
```

Comment:
maybe this can be part of `dockerBuildxSettings` and hence not repeated?

Reply:
Moved that out like suggested.